test/validation.html

```diff
@@ -200,6 +200,15 @@
             expect(tf.invalid).to.be.true;
           });
 
+          it('should force check and change invalid property if `__forceCheckValidity` is set', function() {
```

Comment:
Note, we have to have a correct test for this behavior without using private API in future.

Not a blocker but adding a TODO would not harm.

Reply:
I agree since this is only private now, there should be some note about it being needed for our other components (e.g. date-time-picker) so it doesn't accidentally get removed later. Should probably be converted to protected or public API later (preferably with some better naming or some docs).

Reply:
Note added.